packages/components/src/components/form/SegmentedButton/types.ts

```diff
@@ -0,0 +1,42 @@
+import { ButtonProps as ChakraButtonProps, ResponsiveValue, StyleProps } from '@chakra-ui/react';
+import { FC, ReactElement } from 'react';
+
+export const segmentedButtonVariants = ['primary', 'primary-alt', 'secondary', 'secondary-alt'] as const;
+export type SegmentedButtonVariant = (typeof segmentedButtonVariants)[number];
+export const segmentedButtonSizes = ['xs', 's', 'm', 'l'] as const;
+export type SegmentedButtonSize = (typeof segmentedButtonSizes)[number];
+
+export type Value = string | number;
```

Comment:
this type should be scoped to the component, `Value` too generic to be exported out
```suggestion
export type SegmentedButtonValue = string | number;
```

Reply:
done!